app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -1612,7 +1612,7 @@ The OpenID Connect plugin extends the [Kong Admin API][admin] with a few endpoin
 ### Discovery Cache
 
 When configuring the plugin using `config.issuer`, the plugin will store the fetched discovery
-information to the Kong database, or in the worker memory with Db-less.
+information to the Kong database, or in the worker memory with DB-less. Discovery information remains cached until the OIDC plugin is enabled, disabled, or updated.
```

Comment:
```suggestion
information to the Kong database, or in the worker memory with DB-less. Discovery information is cached except in the following circumstances:
* The plugin can't find a new key, so it retries discovering one.
* JWK information is retained for one rotation before being removed.
* The cache does not poll for new JWK information. This means if a JWK is rolled on the token issuer, the OIDC attempts to find a new key (based on `config.rediscovery_lifetime`) when it cannot find a key. 
* In traditional mode, you can manually delete the issuer cache in the database.
```
@ItsKaleb @bungle Would this work/is this accurate?

Reply:
Hi @cloudjumpercat 

Those changes sound pretty clear to me, though just on the last one, is the cache deletion endpoint not available in Hybrid mode?

Also, as it happens, this conversation became much more relevant this week as my company had a production incident relating to rolling the certificates and the OIDC plugin. I have a new case open with Kong Support to investigate further (00033131). 

Just for visibility, we had the config.rediscovery_lifetime set at 300 seconds, however due to a DNS change our Kong servers were no longer able to reach our IDP. As best as I have been able to tell, it looks like the cache refresh/rediscovery process was failing silently and tokens were being validated based on a very stale cache. I will see what further investigation yields with the support team, but I may have some more edits to suggest based on potential findings.

Thanks.

Reply:
Hi @ItsKaleb ! I'm not sure about the cache deletion endpoint in hybrid mode, I was just going off this comment from @bungle: "On traditional, it is possible to manually delete the issuer cache in db." 

> I may have some more edits to suggest based on potential findings.

Sounds good! I'll leave this PR open.

Reply:
Hi @cloudjumpercat, I've just committed some new updates to the description for rediscovery_lifetime based on feedback from the active support case. It sounds like perhaps there's some contention around the role of this config field? I've included the response from Kong Support in my commit description. Happy to hear if either yourself or @bungle have further comments around that.
Thanks.